server/plugin_release.go

```diff
@@ -543,6 +551,23 @@ func getPluginAsset(ctx context.Context, githubClient *GithubClient, owner, repo
 	}
 }
 
+func markTagAsPreRelease(ctx context.Context, githubClient *GithubClient, owner, repo, tag string, preRelease bool) error {
+	LogInfo("Marking tag as pre release")
+
+	release, _, err := githubClient.Repositories.GetReleaseByTag(ctx, owner, repo, tag)
+	if err != nil {
+		return errors.Wrap(err, "failed to get release by tag")
+	}
+
+	_, _, err = githubClient.Repositories.EditRelease(ctx, owner, repo, release.GetID(), &github.RepositoryRelease{Prerelease: &preRelease})
```

Comment:
should this always be `true`? because releases start out as non-pre-release, we might only need to update it in one case which is to mark a release as pre-release.

Reply:
no, this should not as always be true as rightly said by you. Right now the check for `if preRelease == true` was in the method calling this method but I'll move that here